src/FactSystem/ParameterManager.cc

```diff
@@ -356,7 +356,7 @@ void ParameterManager::_parameterUpdate(int vehicleId, int componentId, QString
             qCDebug(ParameterManagerVerbose1Log) << _logVehiclePrefix(-1) << "Not restarting _waitingParamTimeoutTimer (all requests satisfied)";
         }
     }
-\
+
```

Comment:
this part of the file I like. but you are missing qAsConst in all maps as range-based for will create a copy of the containers.